src/CairoMakie.jl

```diff
@@ -249,17 +209,111 @@ function draw_atomic(scene::Scene, screen::CairoScreen, primitive::Union{Lines,
     ctx = screen.context
     model = primitive[:model][]
     positions = primitive[1][]
+
     isempty(positions) && return
-    N = length(positions)
+
+    projected_positions = project_position.(Ref(scene), positions, Ref(model))
```

Comment:
Test failure happens here...specifically when broadcasting.  Will investigate further.

Reply:
ok this happens because this particular wireframe call results in `positions` being a view of tuples of points

Reply:
ok I've introduced a workaround for that case, now the tests fail only in reference image comparisons (which I guess doesn't mean much as it was bad before?)

Reply:
Ah, that's pretty annoying...guess it makes sense, though.  I guess we should handle that here as well?

Reply:
Yeah, the refimages are expected to fail.  I wonder if we could disable that...